lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1154,13 +1154,8 @@ protected function prepare_links( $post ) {
 					continue;
 				}
 
-				if ( 'post_tag' === $tax ) {
```

Comment:
@jdolan this issue is about using the rest base, so your concern is not specifically here. However, as far as the API exposes data, you'll only ever see `tag`, not a mix - so I don't know why it's a big issue. Tags are not just for posts, so the name is confusing, that's my 2c anyway. This change just fixes one small inconsistency where we didn't use `rest_base` (where we always should be doing).